src/connectedk8s/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+1.8.0
+++++++
+* New api version 2024-07-1-preview added
+* Adding functionality for workload identity feature.
+* Cluster create and update waits for agent state 
+
 1.7.3
```

Comment:
can you please add few tests? also doesnt any of the current tests need updates?